docs/QUICKSTART.md

```diff
@@ -0,0 +1,410 @@
+# Quick-Start Guide for uVisor on mbed OS
```

Comment:
We should probably explicitly mention that this guide's code examples and mbed target descriptions are K64F specific, as we use specific peripherals (SW2, SIM, PORTC, etc.) and extend the existing K64F target to add UVISOR_SUPPORTED.